packages/paypal-commerce-integration/src/paypal-commerce-fastlane/paypal-commerce-fastlane-payment-strategy.ts

```diff
@@ -385,4 +479,25 @@ export default class PaypalCommerceFastlanePaymentStrategy implements PaymentStr
 
         return undefined;
     }
+
+    /**
+     *
+     * PayPal Fastlane experiments handling
+     *
+     */
+    private isPaypalCommerceFastlaneThreeDSAvailable(): boolean {
+        if (this.isFastlaneThreeDSAvailable !== undefined) {
+            return this.isFastlaneThreeDSAvailable;
+        }
+
+        const state = this.paymentIntegrationService.getState();
+        const features = state.getStoreConfigOrThrow().checkoutSettings.features;
+
+        this.isFastlaneThreeDSAvailable = isExperimentEnabled(
+            features,
+            'PROJECT-7080.paypalcommerce_fastlane_three_ds',
+        );
+
+        return this.isFastlaneThreeDSAvailable;
+    }
```

Comment:
```suggestion
    private isPaypalCommerceFastlaneThreeDSAvailable(): boolean {
        const state = this.paymentIntegrationService.getState();
        const features = state.getStoreConfigOrThrow().checkoutSettings.features;

        return isExperimentEnabled(
            features,
            'PROJECT-7080.paypalcommerce_fastlane_three_ds',
        );
    }
```

Reply:
It might be better to simplify this code, since the optimisation proposed in this method is not necessary